lighthouse-cli/test/smokehouse/seo/expectations.js

```diff
@@ -44,4 +47,13 @@ module.exports = [
       }
     }
   },
+  {
+    initialUrl: 'http://localhost:10200/seo/seo-failure-cases-404.html',
+    url: 'http://localhost:10200/seo/seo-failure-cases-404.html',
+    audits: {
+      'http-status-code': {
+        score: false,
+      }
+    }
+  },
```

Comment:
AFAIK current smoketest setup doesn't allow me to easily specify a HTTP code, so I'm pointing smokehouse to non-existent file which ends up returning 404.

This is a hack though. IMO a proper solution for this would be to have an optional `xyz.json` file for each of `xyz-cases.html` files where you can specify what HTTP code and what headers should be returned (we will probably need headers for other audits).

Reply:
FWIW we use query string for some cases like this `delay=2000` delays the response by 2 seconds which we could use for status code, but headers would probably want to go in json direction :)

Reply:
e.g. see the [`?delay=200`](https://github.com/GoogleChrome/lighthouse/blob/b80d906e1d1cd974e068f85362ccf93fb270bf97/lighthouse-cli/test/fixtures/static-server.js#L65-L69) that can be added to delay a response from the server

Reply:
Thanks! I've added `status_code` query param. We will worry about headers later.